### PR TITLE
Create new view for brokers

### PR DESCRIPTION
**Pull request by polskikiel**
**Description**

Changes proposed in this pull request:

- Created a new view for brokers

**Related issue(s)**

#152

### CODE REVIEW

File: brokers/src/components/ServiceBrokers/ServiceBrokersToolbar/ServiceBrokersToolbar.component.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+import { Toolbar } from '@kyma-project/react-components';
+
+const ServiceBrokersToolbar = ({}) => {
+  return (
+    <Toolbar
+      headline="Service Brokers"
+      description="You can view your brokers here"
+    >
+      {/*{serviceBrokersExists ? (*/}
```
Comment (magicmatatjahu) [inline brokers/src/components/ServiceBrokers/ServiceBrokersToolbar/ServiceBrokersToolbar.component.js]: Do you contact with @derberg about enabling/disabling filtering in brokers-ui?

File: brokers/public/config/config.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+var domain = 'kyma.local';
```
[the remark this answers is not in the document]
Comment (polskikiel) [reply brokers/public/config/config.js]: Done

File: brokers/src/components/ServiceBrokers/queries.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import gql from 'graphql-tag';
+
+export const BROKERS_QUERY = gql`
```
[the remark this answers is not in the document]
Comment (polskikiel) [reply brokers/src/components/ServiceBrokers/queries.js]: We already have a view of clusterBrokers. Acceptance criteria also don't say anything about it.

[the remark this answers is not in the document]
Comment (polskikiel) [reply brokers/src/components/ServiceBrokers/queries.js]: Only namespaced as it is in acceptance criteria

File: brokers/README.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+Open the `[http://localhost:8002](http://localhost:8002)` link to view it in the browser.
+
+The page reloads if you make edits.
+If lint errors appear, the console displays them.
+
+### Test the application
+
+Run the `npm test` command to launch the test runner in the interactive watch mode.
+
+See the **Running Tests** section in this [README.md](https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#running-tests) file for more information.
```
[the remark this answers is not in the document]
Comment (polskikiel) [reply brokers/README.md]: Done

File: brokers/OSBAContract.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# Contract with Open Broken Service API
+
+Brokers-ui directly uses the [ui-api-layer](https://github.com/kyma-project/kyma/tree/master/components/ui-api-layer) project to fetch the data. The ui-api-layer fetches the data from Service Brokers using the Service Catalog. The next section explains the mapping of [Service Object](https://github.com/openservicebrokerapi/servicebroker/blob/v2.13/spec.md#catalog-management) from [OSBA](https://openservicebrokerapi.org/) to UI fields.
```
[the remark this answers is not in the document]
Comment (polskikiel) [reply brokers/OSBAContract.md]: ALL DONE 👍

**Pull request merged**